package.json

```diff
@@ -88,11 +88,11 @@
     "eslint-plugin-react": "^7.20.3",
     "glob": "^7.1.6",
     "husky": "^7.0.4",
-    "jest": "^27.4.5",
-    "jest-circus": "^27.4.6",
-    "jest-environment-node": "^27.4.6",
+    "jest": "^28.0.0-alpha.0",
+    "jest-circus": "^28.0.0-alpha.0",
+    "jest-environment-node": "^28.0.0-alpha.0",
     "jest-playwright-preset": "^1.7.0",
-    "jest-runner": "^27.4.6",
+    "jest-runner": "^28.0.0-alpha.0",
```

Comment:
I had to upgrade jest to this new version to import "@brimdata/zealot/node" in the tests. The package.json "exports" field is new and just getting support.